src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -102,9 +107,15 @@
 
     /**
      * The PSS parameter set with all default values
+     * @deprecated This field uses the default values defined in the ASN.1
+     *         encoding in PKCS#1 which may become obsolete as time progresses.
+     *         Thus, it is recommended to create a new {@code PSSParameterSpec}
+     *         object with the desired parameter values using
+     *         {@link #PSSParameterSpec(String, String, AlgorithmParameterSpec, int, int) PSSParameterSpec}.
```

Comment:
Similar rewording suggestion here: 

"This field uses the default values defined in the PKCS#1 standard. Some of these defaults are no longer recommended due to advances in cryptanalysis -- see RFC 8017 for more details. Thus ..."

Reply:
Hmm, RFC 8017 is PKCS#1 v2.2 and I assume you mean PKCS#1 v2.2 with link to RFC 8017.